README.MD

```diff
@@ -183,10 +183,16 @@ sudo tar -xzf bbb_nfs.tar.gz
 
 ## 2.4 Configuring host PC
 
+The purpose of 2.4 is to set up the necessary services to enable the devices to communicate with and
```

Comment:
I think it would be better to say: "In this section, we set up the necessary...." instead of "The purpose of..."